test/test_examples_2d_curved.jl

```diff
@@ -176,6 +176,13 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "struct
       tspan = (0.0, 0.3))
   end
 
+  @trixi_testset "elixir_euler_sedov.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_sedov.jl"),
+      l2   = [3.31660326e-01, 2.24398087e-01, 2.24398087e-01, 1.25705111e+00],
+      linf = [9.06289591e-01, 9.60138464e-01, 9.60138464e-01, 6.12198921e+00],
+      tspan = (0.0, 0.3))
+  end
+
```

Comment:
Question applying to all new tests: How long do they run? How many time steps do they take?

Reply:
p4est_2d_sedov: 2sec, 102 timesteps
p4est_2d_ec: 2sec, 112 timesteps
p4est_3d_sedov: 3sec, 44 timesteps
structured_2d_sedov: 1 sec, 79 timesteps
structured_3d_sedov: 1sec, 35 timesteps
unstructured_2d_sedov: 1 sec, 30 timesteps